src/workerd/server/server.c++

```diff
@@ -1290,6 +1440,12 @@ kj::Own<Server::Service> Server::makeWorker(kj::StringPtr name, config::Worker::
     void reportMetrics(IsolateObserver& isolateMetrics) const override {}
   };
 
+  // Check for the special inspector service.
+  kj::Maybe<InspectorService&> inspectorService;
```

Comment:
I think this is obsolete, since `inspector` itself now has the right type without the need for a downcast?

Reply:
ah right! good catch. Missed this when changing that type.

Reply:
You marked this resolved but this code is still here... were you going to delete it?

Reply:
Well, partially resolved it. What was left wasn't great so cleaned it up more.